saiph/reduction/famd.py

```diff
@@ -532,3 +531,84 @@ def _compute_cos2_single_category(
         np.array(cos2) / summed_weights_without_zeros
     )
     return single_category_cos2
+
+
+def reconstruct_df_from_model(model: Model) -> pd.DataFrame:
+    """Reconstruct a DataFrame from a fitted model.
+
+    Note: if nf < df.shape[1], reconstructed df will not be exactly the same.
+
```

Comment:
Should it raise an error if `model.nf =! nb_dim`? 

by this, we ensure not to encounter some strange behaviors

Reply:
idem for other

Reply:
I think we should not. That's why I added the Note. 
I'll maybe clarify the note saying the more nf is below the number of dimensions, the more the reconstructed df will be modified.
But I am pretty sure raising an error will become a blocker for us in some cases.
I rather prefer to make sure all strange cases are handled.